README.md

```diff
@@ -48,7 +53,17 @@ on any transformations. All QGreenland GeoPackages and GeoTIFFs are projected
 in `EPSG:3413`.
 
 
-# Getting started
+# Level of Support
+
+This repository is fully supported by NSIDC. If you discover any problems or
+bugs, please submit an Issue. If you would like to contribute to this
```

Comment:
Link to issue page here? Also link to our contributing document? Should we move the contributing section here?